knative/eventing
# Set transition time initially on sequence statuses

Fixes #6400

## Proposed Changes
* :bug: Set `LastTransitionTime` when initializing sequence statuses (for subscription and channels)

### Pre-review Checklist

- [ ] **At least 80% unit test coverage**
- [ ] **E2E tests** for any new behavior
- [ ] **Docs PR** for any user-facing impact
- [ ] **Spec PR** for any new API feature
- [ ] **Conformance test** for any change to the spec

**Release Note**
```release-note

```

**How to verify**:
* Create a cluster with kafka
* Apply https://gist.github.com/maschmid/6a8debcc73d8dbdabe4c9c38d7a57517
* Check that the eventing-controller didn't log the error message from #6400 (`k -n knative-eventing logs -f eventing-controller-5dfc99777f-kl2d8 | grep -i invalid`)
* Check that the sequence has a transition time set on its ready `channelStatuses` and `subscriptionStatuses` (`k -n foobar get sequences.flows.knative.dev sequence -o yaml`)
